Makefile

```diff
@@ -30,8 +30,8 @@ all: build unit-test system-test ubuntu-tests
 # 'all-CI' target is used by the scripts/CI.sh that passes appropriate set of
 # ENV variables (from the jenkins job) to run OS (centos, ubuntu etc) and
 # sandbox specific(vagrant, docker-in-docker)
-all-CI: stop clean start
-	make ssh-build
+all-CI: stop clean
+	make checks run-build start ssh-build
```

Comment:
@unclejack compiling the system tests and compiling netplugin are different. I am confused as to what value this change brings. We are not really expecting anyone to push a change that does not even compile. Why add an additional step?